packages/nc-gui/assets/css/global.css

```diff
@@ -9,7 +9,7 @@ html {
 }
 
 body , *  {
-  font-family: 'Inter', 'Source Sans Pro', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, Vazirmatn,
+  font-family: Arial, Vazirmatn, 'Inter', 'Source Sans Pro', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue',
```

Comment:
Based on the direction we are adding the class `rtl` or `ltr`, can you update the font priority order only for `rtl` language? And is there any specific reason to put Arial as the first font family?

```css
body , *  {
  font-family: 'Inter', 'Source Sans Pro', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, Vazirmatn,
}

body.rtl , body.rtl {
 font-family: Vazirmatn, 'Inter', 'Source Sans Pro', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial
}
```

The main reason is to use inter as our default application font.

Reply:
Hi there. The font priority is already set to Vazirmatn for RTL languages but it is not applied everywhere. I thought setting Vazirmatn as the first priority for body would solve the issue.
What is your recommendation?

Reply:
What you sent looks correct.